tasks.py

```diff
@@ -0,0 +1,60 @@
+import csv
+import time
+from urllib.parse import urlparse
+from robocorp.tasks import task
+from robocorp import browser
+
+
+@task
+def take_website_screenshots():
+    """Take screenshots of all websites found in websites.csv file."""
+    with open("websites.csv", mode="r") as csv_file:
+        csv_reader = csv.DictReader(csv_file)
+        websites = list(csv_reader)
+
+    for website in websites:
+        browser.goto(website["url"])
+        page = browser.page()
+
+        accept_cookies_and_consents(website)
+
+        # some websites have animations on cookies and consents so wait for that to disappear
+        time.sleep(1)
+
+        domain = get_domain_name(website["url"])
+        page.screenshot(path=f"output/{domain}.png", full_page=True)
```

Comment:
🐛 We usually define a constant for the output dir:
```python
OUTPUT_DIR = Path(os.environ.get("ROBOT_ARTIFACTS", "output"))
```
and then you can do:
```suggestion
        page.screenshot(path=str(OUTPUT_DIR / f"{domain}.png"), full_page=True)
```